pyupgrade/_plugins/unittest_aliases.py

```diff
@@ -12,50 +12,67 @@
 from pyupgrade._data import TokenFunc
 from pyupgrade._token_helpers import replace_name
 
-
 METHOD_MAPPING_PY27 = {
-    'assertEquals': 'assertEqual',
-    'failUnlessEqual': 'assertEqual',
-    'failIfEqual': 'assertNotEqual',
-    'failUnless': 'assertTrue',
-    'assert_': 'assertTrue',
-    'failIf': 'assertFalse',
-    'failUnlessRaises': 'assertRaises',
-    'failUnlessAlmostEqual': 'assertAlmostEqual',
-    'failIfAlmostEqual': 'assertNotAlmostEqual',
+    'self': {
+        'assertEquals': 'assertEqual',
+        'failUnlessEqual': 'assertEqual',
+        'failIfEqual': 'assertNotEqual',
+        'failUnless': 'assertTrue',
+        'assert_': 'assertTrue',
+        'failIf': 'assertFalse',
+        'failUnlessRaises': 'assertRaises',
+        'failUnlessAlmostEqual': 'assertAlmostEqual',
+        'failIfAlmostEqual': 'assertNotAlmostEqual',
+    },
+    'unittest': {
+        'findTestCases': 'defaultTestLoader.loadTestsFromModule',
+        'makeSuite': 'defaultTestLoader.loadTestsFromTestCase',
+        'getTestCaseNames': 'defaultTestLoader.getTestCaseNames',
+    },
 }
 
 METHOD_MAPPING_PY35_PLUS = {
-    **METHOD_MAPPING_PY27,
-    'assertNotEquals': 'assertNotEqual',
-    'assertAlmostEquals': 'assertAlmostEqual',
-    'assertNotAlmostEquals': 'assertNotAlmostEqual',
-    'assertRegexpMatches': 'assertRegex',
-    'assertNotRegexpMatches': 'assertNotRegex',
-    'assertRaisesRegexp': 'assertRaisesRegex',
+    'self': {
+        **METHOD_MAPPING_PY27['self'],
+        'assertNotEquals': 'assertNotEqual',
+        'assertAlmostEquals': 'assertAlmostEqual',
+        'assertNotAlmostEquals': 'assertNotAlmostEqual',
+        'assertRegexpMatches': 'assertRegex',
+        'assertNotRegexpMatches': 'assertNotRegex',
+        'assertRaisesRegexp': 'assertRaisesRegex',
+    },
+    'unittest': {
+        **METHOD_MAPPING_PY27['unittest'],
+    },
 }
 
 
 @register(ast.Call)
 def visit_Call(
-        state: State,
-        node: ast.Call,
-        parent: ast.AST,
+    state: State,
+    node: ast.Call,
+    parent: ast.AST,
 ) -> Iterable[tuple[Offset, TokenFunc]]:
```

Comment:
generally shouldn't touch formatting unrelated to your change -- also I prefer these double-indented to differentiate from the code

Reply:
I must've accidentally ran `black`. Will revert.